src/model/import.ts

```diff
@@ -30,12 +30,10 @@ export class ScannedImport implements Resolvable {
   type: 'html-import'|'html-script'|'html-style'|'js-import'|string;
 
   /**
-   * URL of the import, relative to the document containing the import.
+   * URL of the import, relative to the base directory.
```

Comment:
+1, but punting. This PR doesn't change this behavior I don't think, it just documents the current behavior.